config_local.js.example

```diff
@@ -5,3 +5,11 @@
 config.stats.enabled = true;
 config.stats.summary = true;
 config.profiler.enabled = false;
+
+global.mainLocal = function() {
```

Comment:
I wouldn't put logic into the config files.
IMHO the logic can be moved to `src/main.js` after `brain.stats.updateCpuStats();`, with a feature flag in the config `config.pixel.enabled`.

Instead of checking the bucked you could check for CPU still available after the rest of the tick was already executed.

Reply:
Thx for your feedback.

I think you're right the config shouldn't include code!
I've got some custom logic that prints links to the rooms with stats and the current queue. This way I've got an overview over all rooms in the console. That's why I would like to have an entrypoint into the main function.
What do you think about a `main_local.js` instead of the mainLocal from `config_local.js`?

The current logic for pixel generation could be extended with another condition to check if there's still CPU left and with an config option to enable pixel generation.
I think thats a good idea. The pixel generation is just using the bucket and should include PIXEL_CPU_COST + 1000 so that there is always at least 1k cpu left.

Reply:
Regarding: `main_local`: If it helps, include it.
But, if you have logging or logical which helps, why not include it directly. E.g. there are the `config.debug` options that make the logging more verbose. If you need additional information, make a config flag, default to `false`. In your local config enable it. This way others can use that, too.

regarding pixel: Sounds good,

Reply:
Added configuration for `pixels` and `tickSummary` in latest commit.
Added `main_local` in latest commit.

Any suggestions regarding the latest changes?

Reply:
Add some comments, basically I'm fine.

Reply:
I will polish this the next few days then we can merge it 👍

Reply:
Sounds good, can you please squash your changes to a single commit when you are satisfied?!